awx/main/analytics/collectors.py

```diff
@@ -0,0 +1,147 @@
+        'ansible_version': get_ansible_version(),
+        'license_type': license_info.get('license_type', 'UNLICENSED'),
+        'free_instances': license_info.get('free instances', 0),
+        'license_expiry': license_info.get('time_remaining', 0),
+        'authentication_backends': settings.AUTHENTICATION_BACKENDS,
+        'logging_aggregators': settings.LOG_AGGREGATOR_LOGGERS
+    }
+
+@register('counts')
+def counts(since):
+    counts = {}
+    for cls in (models.Organization, models.Team, models.User,
+                models.Inventory, models.Credential, models.Project,
+                models.JobTemplate, models.WorkflowJobTemplate, models.Host,
+                models.Schedule, models.CustomInventoryScript,
+                models.NotificationTemplate):
+        counts[camelcase_to_underscore(cls.__name__)] = cls.objects.count()
+
+    venvs = get_custom_venv_choices()
+    counts['custom_virtualenvs'] = len([
+        v for v in venvs
+        if os.path.basename(v.rstrip('/')) != 'ansible'
+    ])
+
+    counts['active_host_count'] = models.Host.objects.active_count()
+    counts['smart_inventories'] = models.Inventory.objects.filter(kind='smart').count()
+    counts['normal_inventories'] = models.Inventory.objects.filter(kind='').count()
+
+    active_sessions = Session.objects.filter(expire_date__gte=now()).count()
+    api_sessions = models.UserSessionMembership.objects.select_related('session').filter(session__expire_date__gte=now()).count()
+    channels_sessions = active_sessions - api_sessions
+    counts['active_sessions'] = active_sessions
+    counts['active_api_sessions'] = api_sessions
+    counts['active_channels_sessions'] = channels_sessions
+    counts['running_jobs'] = models.Job.objects.filter(status='running').count()
+    return counts
+    
+    
+@register('org_counts')
+def org_counts(since):
+    counts = {}
+    for org in models.Organization.objects.annotate(num_users=Count('member_role__members', distinct=True), 
+                                                    num_teams=Count('teams', distinct=True)):  # Use .values to make a dict of only the fields we can about where
+        counts[org.id] = {'name': org.name,
+                            'users': org.num_users,
+                            'teams': org.num_teams
+                            }
+    return counts
+    
+    
+@register('cred_type_counts')
+def cred_type_counts(since):
+    counts = {}
+    for cred_type in models.CredentialType.objects.annotate(num_credentials=Count('credentials', distinct=True)):  
+        counts[cred_type.id] = {'name': cred_type.name,
+                                  'credential_count': cred_type.num_credentials
+                                }
+    return counts
+    
+    
+@register('inventory_counts')
+def inventory_counts(since):
+    counts = {}
+    from django.db.models import Count
+    for inv in models.Inventory.objects.annotate(num_sources=Count('inventory_sources', distinct=True), 
```

Comment:
I found a way to simplify this:

```
list(Inventory.objects.filter(kind='smart').values('id').annotate(num_sources=Count('inventory_sources'), 
                                            num_hosts=Count('hosts')).values('id', 'kind', 'name', 'num_sources', 'num_hosts'))[0]
```

Unfortunately though, smart inventories use a different manager meaning all of the host counts for smart_inventories are zero unless I grab each smart_inventory object, so I'll have to do the smart_inventories in a separate query.